--- 00hello/main.go	2022-01-17 15:46:27.620110890 +0900
+++ 01json-tag/main.go	2022-01-29 13:37:34.528688891 +0900
@@ -8,8 +8,8 @@
 )
 
 type Options struct {
-	Name    string `flag:"name" help:"name of greeting"`
-	Verbose bool   `flag:"verbose" short:"v"`
+	Name    string `json:"name"`
+	Verbose bool   `json:"verbose" short:"v"`
 }
 
 func main() {
@@ -18,6 +18,7 @@
 	b := structflag.NewBuilder()
 	b.Name = "hello"
 	b.EnvPrefix = "X_"
+	b.FlagnameTags = append(b.FlagnameTags, "json")
 
 	fs := b.Build(options)
 	fs.Parse(os.Args[1:])
