--- 00hello/main.go	2022-08-05 19:47:52.713702572 +0900
+++ 01json-tag/main.go	2022-08-05 19:48:54.083703727 +0900
@@ -7,8 +7,10 @@
 )
 
 type Options struct {
-	Name    string `flag:"name" help:"name of greeting"`
-	Verbose bool   `flag:"verbose" short:"v"`
+	Name        string `json:"name"`
+	Verbose     bool   `json:"verbose" short:"v"`
+	Ignored     bool   `json:"ignored" flag:"-"`
+	AnotherName string `json:"anotherName" flag:"another-name"`
 }
 
 func main() {
@@ -17,6 +19,7 @@
 	flagstruct.Parse(options, func(b *flagstruct.Builder) {
 		b.Name = "hello"
 		b.EnvPrefix = "X_"
+		b.FlagnameTags = append(b.FlagnameTags, "json")
 	})
 	fmt.Printf("parsed: %#+v\n", options)
 }
